docs/faq/admin.txt

```diff
@@ -97,11 +97,15 @@ like to make should be possible by editing the stylesheet.
 What browsers are supported for using the admin?
 ================================================
 
-The admin provides a fully-functional experience to `YUI's A-grade`_ browsers,
-with the notable exception of IE6, which is not supported.
+The admin provides a fully-functional experience to the recent versions of
+modern, web standards compliant browsers. On desktop this means Edge, Firefox,
+Chrome, Safari, and others.
 
-There *may* be minor stylistic differences between supported browsers—for
-example, some browsers may not support rounded corners. These are considered
-acceptable variations in rendering.
+On smaller, mobile and tablet devices, the admin provides a responsive
+experience for web standards compliant browsers. This includes the major
+browsers on both Android and iOS.
 
-.. _YUI's A-grade: https://github.com/yui/yui3/wiki/Graded-Browser-Support
+Depending on feature support there *may* be minor stylistic differences between
+browsers. These are considered acceptable variations in rendering.
+
+Microsoft's legacy Internet Explorer is not supported.
```

Comment:
I would add a release note, e.g.
```
``contrib.admin`` browser support
---------------------------------

The admin no longer supports Internet Explorer, as this browser have reached
end-of-life.
```

Reply:
OK, I'll add that. 

(Although we're already not compatible... ticket-31309)

Reply:
We need a different wording, IE11 is not recommended but it's still supported.

Reply:
Take a look.